src/app/leverage/components/leverage-widget/index.tsx

```diff
@@ -76,7 +78,13 @@ export function LeverageWidget() {
     outputToken,
     inputValue,
   )
-  const { buttonLabel, isDisabled } = useTradeButton(buttonState)
+  const { buttonLabel: generatedButtonLabel, isDisabled } =
+    useTradeButton(buttonState)
+
+  const buttonLabel = useMemo(() => {
+    if (generatedButtonLabel === 'Swap') return 'Review Transaction'
```

Comment:
nit: could update 'Swap' to be part of an enum in the future in case we change the label